query-engine/connector-test-kit-rs/query-engine-tests/tests/new/error.rs

```diff
@@ -0,0 +1,61 @@
+use query_engine_tests::*;
+
+#[test_suite(schema(schema))]
+mod errors {
+    use indoc::indoc;
+    use query_engine_tests::run_query;
+
+    fn schema() -> String {
+        let schema = indoc! {
+            r#"model Album {
+                    id Int @id @unique
```

Comment:
This test will fail on mongo. The `#id(id, Int, @id, @unique)` syntax is there to render the ID for connectors as needed.